CHANGELOG.md

```diff
@@ -8,6 +8,7 @@
 
 - **Improved wizard responsive design** by using percentage width.
 - **Fixed user-icon size not working when placed in icon button.**
+- **Fixed `DEPRECATION WARNING: Using / for division is deprecated and will be removed in Dart Sass 2.0.0.`.** when using scss files in project with Dart Sass 1.32.0 or later.
```

Comment:
Create a new section for this like so:

https://github.com/iTwin/iTwinUI-react/blob/272fd7fc33b1302d8aaa9a8c09bd4a5e1a623754/CONTRIBUTING.md#L153-L157

Also could reword it to focus on the :
"Fixed Sass deprecation warning about division operator** by replacing `/` with `*`.

Reply:
good point - will fix

Reply:
updated